Feature: Build a workshop / sandbox quickstart to test out Fusion in Coalesce

coapacetic opened the pull request:
## What are you changing in this pull request and why?
Very lightweight quickstart to access an environment for seeing Fusion development and deployment in 

## Checklist
- [ ] Needs technical review
- [ ] Change base branch
- [ ] Merge on October 13th
- [X] The changes in this PR meet the [docs style guide/fundamentals](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) required for all content.
- [X] Applied the proper versioning rules if the content is for specific dbt version(s): ([version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)

ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [X] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

coapacetic committed:
add in QS md and images

coapacetic committed:
improve with local development to get preview working

coapacetic committed:
Merge branch 'current' into feat/quick-start-for-fusion-sandbox

kbrock91 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,92 @@
+
+6. Click **Login**  
+
+## Section 2 - Developer Productivity with Fusion
```
can we change to ":" for consistency across sections

kbrock91 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,92 @@
+
+    Please note: any file edits stay local to your Studio IDE; no feature branches are permitted for this sandbox account.
+
+## Section 3 - State-Aware Orchestration
```
can we change to ":" for consistency across sections

kbrock91 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,92 @@
+
+<Lightbox src="/img/guides/coalesce-fusion-qs/models-reused.png" width="90%" title="💡 State aware orchestration ensures only changed data is processed." />
+
+## Section 4 - Learn More
```
can we change to ":" for consistency across sections

kbrock91 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,92 @@
+
+## Section 4 - Learn More
+
+Stop by the **dbt Booth in the Discovery Hall** to learn more about the Fusion-powered features mentioned in the keynote, witness the magic firsthand at any of our demo pods, and dive deeper into use cases, success stories, and migration details through ongoing lightning talks.
```
do we want to add any other CTAs?

coapacetic committed:
implement KB feedback, add Wistia video

coapacetic committed:
quick polish with KB

mirnawong1 committed:
Update coalesce-fusion-platform-qs.md

made direct edits to:
- use constants for the feature name 
- update casing to reflect official feature/product name (platform is lowercased for example)

mirnawong1 committed:
Update coalesce-fusion-platform-qs.md

fix typos

mirnawong1 committed:
Update coalesce-fusion-platform-qs.md

mirnawong1 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,112 @@
+
+<Lightbox src="/img/guides/coalesce-fusion-qs/sql-error.png" width="90%" title="Fusion can detect your SQL errors in real-time while you code and before you send it to your data platform." />
+
+    Please note: any file edits stay local to your Studio IDE; no feature branches are permitted for this sandbox account.
```
```suggestion
    Please note: any file edits stay local to your <Constant name="cloud_ide"/>; no feature branches are permitted for this sandbox account.
```

mirnawong1 reviewed (approved):
looks awesome! made some small editorial tweaks directly - good to go for oct 13th if you're both ok with it!

mirnawong1 commented on website/docs/guides/coalesce-fusion-platform-qs.md:
```diff
@@ -0,0 +1,112 @@
+Learn more about the <Constant name="fusion_engine"/> in the docs: [About Fusion](https://docs.getdbt.com/docs/fusion/about-fusion)
+
+Continue learning dbt at [learn.getdbt.com](http://learn.getdbt.com)
+
```
sugg also linking to coalesce conference page
```suggestion
Learn about [Coalesce](https://coalesce.getdbt.com/event/21662b38-2c17-4c10-9dd7-964fd652ab44/summary) &mdash; the premier analytics engineering conference where data leaders and practitioners reimagine tools and reshape the future of data and AI.
```

coapacetic committed:
Update website/docs/guides/coalesce-fusion-platform-qs.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

coapacetic committed:
Update website/docs/guides/coalesce-fusion-platform-qs.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

coapacetic committed:
Merge branch 'current' into feat/quick-start-for-fusion-sandbox

coapacetic committed:
Merge branch 'feat/quick-start-for-fusion-sandbox' of github.com:dbt-labs/docs.getdbt.com into feat/quick-start-for-fusion-sandbox
